website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md

```diff
@@ -398,12 +398,12 @@ Later, you can connect your business intelligence (BI) tools to these views and
 ## Connect to multiple data sources
 This quickstart focuses on using dbt Cloud to run models against a data lake (S3) by using Starburst Galaxy as the query engine. In most real world scenarios, the data that is needed for running models is actually spread across multiple data sources and is stored in a variety of formats. With Starburst Galaxy, Starburst Enterprise, and Trino, you can run your models on any of the data you need, no matter where it is stored.
 
-If you want to try this out, you can use the [Starburst Galaxy docs](https://docs.starburst.io/starburst-galaxy/catalogs/) to add more data sources and load the Jaffle Shop data into the source you select. Then, extend your models to query the new data source and the data source you created in this quickstart. 
+If you want to try this out, you can refer to the [Starburst Galaxy docs](https://docs.starburst.io/starburst-galaxy/catalogs/) to add more data sources and load the Jaffle Shop data into the source you select. Then, extend your models to query the new data source and the data source you created in this quickstart.
 
 ## Related content
 - Learn more with the [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) course for free
 - [dbt Courses](https://courses.getdbt.com/collections)
 - [dbt Cloud CI job](/docs/deploy/cloud-ci-job)
 - [Job notifications](/docs/deploy/job-notifications)
 - [Source freshness](/docs/deploy/source-freshness)
-- [SQL basics for Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/tutorials/learn-basics.html)
\ No newline at end of file
+- [SQL overview for Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/sql/index.html)
```

Comment:
The overview contains a link to the tutorial and much more other useful links and info